<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferActionMenu</class>
 <widget class="QDialog" name="TransferActionMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="1">
      <widget class="QPushButton" name="btAmAttnXfer">
       <property name="text">
        <string>Normal</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btAmPound">
       <property name="accessibleName">
        <string>0x23</string>
       </property>
       <property name="text">
        <string>#</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btAm4">
       <property name="accessibleName">
        <string>0x34</string>
       </property>
       <property name="text">
        <string>4 ghi</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btAm9">
       <property name="accessibleName">
        <string>0x39</string>
       </property>
       <property name="text">
        <string>9 wxyz</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="btAmBlndXfer">
       <property name="text">
        <string>Blind</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btAm5">
       <property name="accessibleName">
        <string>0x35</string>
       </property>
       <property name="text">
        <string>5 jkl</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btAm6">
       <property name="accessibleName">
        <string>0x36</string>
       </property>
       <property name="text">
        <string>6 mno</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btAm1">
       <property name="accessibleName">
        <string>0x31</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btAm7">
       <property name="accessibleName">
        <string>0x37</string>
       </property>
       <property name="text">
        <string>7 pqrs</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btAm2">
       <property name="accessibleName">
        <string>0x32</string>
       </property>
       <property name="text">
        <string>2 abc</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btAmStar">
       <property name="accessibleName">
        <string>0x2a</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btAm3">
       <property name="accessibleName">
        <string>0x33</string>
       </property>
       <property name="text">
        <string>3 def</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btAm8">
       <property name="accessibleName">
        <string>0x38</string>
       </property>
       <property name="text">
        <string>8 tuv</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="btAmCancelXfer">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btAm0">
       <property name="accessibleName">
        <string>0x30</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="lbTransferExt">
       <property name="text">
        <string>Extention</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btAmBs">
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btAm0</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAm1</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAm2</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAm3</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAm5</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAm6</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAm7</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAm8</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAm9</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAm4</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAmStar</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAmPound</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>digitClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAmAttnXfer</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>onControlBtnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAmBlndXfer</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>onControlBtnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAmCancelXfer</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>onControlBtnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAmBs</sender>
   <signal>clicked()</signal>
   <receiver>TransferActionMenu</receiver>
   <slot>onBsBtnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>digitClick()</slot>
  <slot>onControlBtnClick()</slot>
  <slot>onBsBtnClick()</slot>
 </slots>
</ui>
